### ISSUE TITLE
wordsmithing 6

### ISSUE THREAD

**Issue by jeankaplansky** [w3c/epubweb#20]
"Book publishers are currently investing in the development of technical expertise in Web technologies; an expertise that is not part of the “traditional” work of publishers. Whilst gaining some level of expertise is important, the lack of synergy between trade publishers and the Web application developers’ community also means unnecessary duplications, i.e., unnecessary investments.

Being able to collaborate more closely with the Web content development community, with all its inventiveness and agility, would have major benefit for publishers. Through the usage of a universal, interoperable format, it will let publishers concentrate on engaging with the content authors to produce finished and high quality content, and rely on the Web content community to deal with the more sophisticated technical issues (such as intricacies of CSS or of SVG). Possible future content formats on the Web (e.g., 3D rendering) or various interactive Web programs (e.g., visualization tools like D3) will also naturally flow into the realm of publishers through EPUB-WEB and hence increase the publishers’ possibilities. This is especially true for areas of publishing that are traditionally at the leading edge of electronic content, such as STM and educational publishing (see also the section on scholarly publishing below). Basically, a more converged platform, instead of parallel universes, will support more tools and services and a much larger population of trained practitioners."

to

"Book publishers are investing in the development of technical expertise in web technologies. People with technical expertise by anyone other than information technology departments were not previously required in traditional publishing workflows. However, while gaining some understanding of technical topics is important to new and future publishing workflows, the lack of communication between the trade publishers and web application developer communities is resulting in unnecessary duplication and investments in effort.

Collaboration between the web content development and publishing communities will result in major benefits to publishers. Adopting a universal and interoperable format means publishers can concentrate on engaging content authors in the production of high quality content. The web content development community can be relied on to deal with sophisticated technical issues (e.g., CSS, SVG). Potential future web content formats (e.g., 3D rendering) and various interactive web programs (e.g., visualization tools like D3 [Jean note: better be prepared to tell people what D3 is...]) will naturally flow into the publishing realm through EPUB-WEB, hence increasing publishers' opportunities to sell new content products across the board.

Realizing new opportunities is a reality for publishers traditionally considered to be on the leading edge of technological advances in working with content. These publishers include STM and educational publishing houses, as well as scholarly and journal publishing organizations (see the section on scholarly publishing in this document).

A converged platform will support more tools and services and a much larger population of trained practitioners compared to the current state of working in parallel universes."

**Comment by iherman** [comment]
Wow. That was a big one:-) Added, also added a link to D3.
